approve.py

```diff
@@ -30,13 +30,9 @@
     user_name: \n\
     group_name: {group_name}
     callink_oid: {callink_oid}
-    signatory: \n\
     email: {email}
 
     # Please ensure that:
-    #  * Person requesting account is signatory of group
-    #    - Look up the signatory's CalNet UID on directory.berkeley.edu
-    #    - Use `signat <uid>` to list groups they are a signatory for
```

Comment:
You'll want to keep this note. The only thing we don't use anymore is the first thing you removed.